scripts/python/qa-scripts/createVCAAggregate.py

```diff
@@ -0,0 +1,523 @@
+                    vca_fn = self.vcasFileList[filesIdx]
+                    vca_filename = os.path.basename(vca_fn.replace('\\',os.sep))
+                    single_vca = next((item for item in self.vcas if item['vca_file'] == vca_filename), None)
+                    if (integrity is False):
+                        print("{} failed to pass the integrity test at id {}".format(vca_fn, id))
+                    if integrity:
+                        bad = self.badFeedback(locAss)
+                        good = self.goodFeedback(locAss)
+                        excellent = self.excellentFeedback(locAss)
+                        if (self.isVCAfeedbackValid(locAss, bad, good, excellent)):
+                            if (bad or good or excellent):
+                                reviews_num_col = "No. of Reviews"
+                                if locAss[self.opt.challengeCol] in self.dChallenges:
+                                    reviews_num_col = reviews_num_col + " " + locAss[self.opt.challengeCol]
+                                if reviews_num_col in single_vca:
+                                    single_vca[reviews_num_col] = single_vca[reviews_num_col] + 1
+                                else:
+                                    single_vca[reviews_num_col] = 1
+                                self.dfVca.loc[id, self.opt.noVCAReviewsCol] = self.dfVca.loc[id, self.opt.noVCAReviewsCol] + 1
+                                # Append the single review to the vcaMerged file
+                                toBeMergedAssessment = locAss.copy()
+                                toBeMergedAssessment['id'] = id
+                                toBeMergedAssessment['name'] = single_vca['name']
+                                self.vcaMerged = self.vcaMerged.append(toBeMergedAssessment)
+
+                            for col in self.allColumns:
+                                colVal = self.checkIfMarked(locAss, col)
+                                if (colVal > 0):
+                                    self.dfVca.loc[id, col] = self.dfVca.loc[id, col] + colVal
+
+            (bad, good, excellent) = self.calculateOutcome(self.dfVca.loc[id])
+            self.dfVca.loc[id, 'Result ' + self.opt.notValidCol] = bad
+            self.dfVca.loc[id, 'Result ' + self.opt.goodCol] = good
+            self.dfVca.loc[id, 'Result ' + self.opt.excellentCol] = excellent
+
+        vcaAggregatedAssessments = pd.DataFrame(self.dfVca).copy()
+        vcaAggregatedAssessments.fillna('', inplace=True)
+        vcaAggregatedAssessments[self.opt.assessmentsIdCol] = vcaAggregatedAssessments.index
+
+        aggregatedAssessments = pd.DataFrame(self.dfVca).copy()
+        aggregatedAssessments.fillna('', inplace=True)
+        aggregatedAssessments[self.opt.assessmentsIdCol] = aggregatedAssessments.index
+        aggregatedAssessments[self.opt.excellentCol] = aggregatedAssessments['Result ' + self.opt.excellentCol]
```

Comment:
why don't we directly used the `self.opt.excellentCol`  from the beginning?

Reply:
We can't because `self.opt.excellentCol` contains the count (that will be published for transparency), and `Result + ...` the outcome of the consensus based on the counters.
In some sheets published from these data both types are needed at the same time.